osafw-app/App_Code/controllers/MyLists.cs

```diff
@@ -40,9 +47,18 @@ public override Hashtable initFilter(string session_key = null)
             return this.list_filter;
         }
 
+        public override void setListSearchStatus()
+        {
+            if (!string.IsNullOrEmpty((string)list_filter["status"]))
+            {
+                this.list_where += " and status=@status";
+                this.list_where_params["@status"] = list_filter["status"];
+            }
```

Comment:
if you removed status<>127 from setListSearch() then it should be here in else condition